pipes/WDL/tasks/tasks_16S_amplicon.wdl

```diff
@@ -1,19 +1,21 @@
 version 1.0 
-##FN 
 #qiime2022.8 version
+#11.16.22
 
 #Part 1A | Step 1: BAM -> QZA
 #qiime import bam-(fastq)-> qza 
 task qiime_import_from_bam {
 
-    meta{
-        description: "Alignments in the SAM format, including sorting, merging, indexing and generating alignments in a per-position format."
+    meta {
+        description: "Parsing demultiplexed fastq BAM files into qiime readable files."
     }
 
     input { 
         File    reads_bam
         String  sample_name = basename(reads_bam, '.bam')
-
+        Int     memory_mb = 2000
```

Comment:
unless you're passing `-XmX` flags to Java, maybe keep it simple and specify memory in GB (most of our WDL tasks specify in GB) -- comment applies to all wdl tasks